client/src/containers/ConnectedHeader/tests/index.test.tsx

```diff
@@ -47,4 +45,17 @@ describe('components/ConnectedHeader', () => {
     expect(wrapper.find('Header').props()).toMatchSnapshot();
     wrapper.unmount();
   });
+
+  it('renders the Language switcher', () => {
+    jest.mock('../../../configs/env');
```

Comment:
The envs are already mocked at the top. Do we need to re-mock here?

Reply:
oh sorry my bad didnt saw the import at top